composer.json

```diff
@@ -54,7 +54,8 @@
     ],
     "require": {
         "php": ">=7.4 <8.2",
-        "atk4/data": "dev-develop"
+        "atk4/data": "dev-develop",
+        "nyholm/psr7": "^1.4"
     },
     "require-release": {
```

Comment:
make sure `nyholm/psr7` is added here as well, 1:1

Reply:
i dunno why is missing, i do a rollback, after some changes to merge App.php diff correctly and push was approved.